### DISCUSSION TITLE
Capturing updates to an adaptive VDOM

### DISCUSSION THREAD

**Discussion by jbeeko** [fsprojects/FSharp.Data.Adaptive#106] [General]
In his 2020 Oslo talk Don Syme mentions how listeners can be attached to an adaptive VDOM and capture changes ([https://youtu.be/us4dp7Ksly0?t=2346…](https://t.co/3CEBSao7i2) ). Are there any examples in the Repo demonstrating this? It was suggested that [Georg Haaser](https://github.com/krauthaufen) may have created a demo of this.

**Comment by krauthaufen** [comment]
Hey, actually there are several demos/libs
One in fable is [here](https://github.com/krauthaufen/Fable.Elmish.Adaptive)
And we have two implementations that work server-side. The currently used one is in
https://github.com/aardvark-platform/aardvark.media/tree/master/src/Aardvark.UI
and a new sektch is in https://github.com/aardvark-community/aardvark.dom
